libs/dh/wholesale/feature-calculations/src/lib/filters/filters.component.ts

```diff
@@ -120,7 +120,14 @@ export class DhCalculationsFiltersComponent implements OnInit {
 
   _processTypeOptions = this.transloco
     .selectTranslateObject('wholesale.calculations.processTypes')
-    .pipe(map((t) => processTypes.map((k) => ({ displayValue: t[k], value: k }))));
+    .pipe(
+      map((t) =>
```

Comment:
overvej at omdøbe t til translations og k til value

Reply:
Agreed, should be fixed now